src/schema/basic_slots.yaml

```diff
@@ -413,7 +413,7 @@ slots:
     description: >
       The type of analyte(s) that were measured in the data generation process and analyzed
         in the Workflow Chain
-    range: AnalyteCategoryEnum
+
```

Comment:
👍 - so for now, we fall back to "string" for the range of `analyte_category` if not otherwise defined in `slot_usage`.